Documents/Automation/EncryptRootVolume/Design/Design.md

```diff
@@ -0,0 +1,28 @@
+# Encrypt EBS root volume
+
+## Notes
+
+Encrypts the root volume of an EC2 instance.  This will be a replace operation and not an in-line encryption operation.
+
+## Document Design
+
+Refer to schema.json
+
+Document Steps:
+1. aws:npark-encryptrootvolume - Execute CloudFormation Template to attach the volume.
```

Comment:
These are not the document steps. You need to list all the actions used, not the input parameters of your document. The input parameters should have their own section.

Reply:
Correct.  I've updated the Design.md.  Please review to see whether it is to your liking.